utils/sclang.in

```diff
@@ -135,6 +135,9 @@ if ($souper) {
     if (exists $ENV{"SOUPER_STATS"}) {
         push @ARGV, ("-mllvm", "-stats");
     }
+    if (exists $ENV{"SOUPER_USE_HARVEST_KNOWN_BITS"}) {
```

Comment:
one last thing: this should be SOUPER_HARVEST_KNOWN_BITS not SOUPER_USE_HARVEST_KNOWN_BITS